posts/inside-rust/2023-09-19-crates-io-usage-policy-rfc.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: crates.io Policy Update RFC
+author: Tobias Bieniek
+team: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
+---
+
+Around the end of July the crates.io team opened an [RFC](https://github.com/rust-lang/rfcs/pull/3463) to update the current crates.io usage policies. This has caused a lot of controversial discussion and most of the suggested improvements have since been integrated into the proposal.
```

Comment:
The one thing that might be missing here is a brief explanation of the motivation for the policy update in the first place: the tl;dr below summarises the controversial parts of the changes, but not _why_ they're being proposed in the first place.

Here's an idea for framing (I also tinkered with the "controversial" part just a touch — readers can decide for themselves how they feel, I think):

```suggestion
Around the end of July the crates.io team opened an [RFC](https://github.com/rust-lang/rfcs/pull/3463) to update the current crates.io usage policies. This update addresses operational concerns that have arisen since the last significant update in 2017, particularly related to name squatting and spam. This has caused considerable discussion, and most of the suggested improvements have since been integrated into the proposal.
```